src/Components/WebAssembly/testassets/HostedInAspNet.Server/Properties/launchSettings.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
I'm committing the `launchSettings.json` files for these sample projects because otherwise you have to keep adding the `inspectUri` setting to try out debugging.

Reply:
That sounds fine